test/e2e/metrics_test.go

```diff
@@ -57,13 +69,25 @@ func TestCatalogdMetricsExportedEndpoint(t *testing.T) {
 		"https://catalogd-service.NAMESPACE.svc.cluster.local:7443/metrics",
 	)
 
-	config.run()
+	mtc.run()
+}
+
+func findK8sClient(t *testing.T) (kubernetes.Interface, *rest.Config) {
+	cfg, err := config.GetConfig()
+	require.NoError(t, err, "Failed to get Kubernetes config")
```

Comment:
Do we already have `cfg` from `TestMain` as well?

Reply:
yah, same, fixed in recent commit